app/views/layouts/_content.html.haml

```diff
@@ -63,6 +63,9 @@
             = render :partial => "layouts/listnav"
           = yield :left
         #custom_left_cell
+- elsif @angular_app_name
```

Comment:
@AparnaKarve, @karelhala .. feels like `@angular_app_name` + these `_content` changes are mostly doing the same thing as Aparna's  `def angular_app?` + `layouts/application` in https://github.com/ManageIQ/manageiq/pull/6293/files .. not quite sure yet which one is better, but we should probably settle for one..